jdaviz/configs/imviz/plugins/coords_info/coords_info.py

```diff
@@ -469,7 +469,7 @@ def _copy_axes_to_spectral():
                     sp = self.app._get_object_cache[cache_key]
                 else:
                     sp = self._specviz_helper.get_data(data_label=data_label,
-                                                       subset_to_apply=subset_label)
+                                                       spectral_subset=subset_label)
```

Comment:
Are you sure this is trying to get the spectral subset and not a collapsed spatial subset?

Reply:
Regardless, the arg name has changed, so that necessitates the change here. Unless this wasn't considered when the decision was made to rename the arg and we need to rethink this?

Reply:
@kecnry did this part so he would know best.

Reply:
yes, I believe we want spatial here (you can test it by seeing if mouseover on the live-collapsed spectrum still works)

Reply:
Done! Fixed in https://github.com/spacetelescope/jdaviz/pull/2242/commits/5a9193e2388c12638e12e75b2f93b7a1d2e4ee92 and verified the mouseover plugin works in imviz and cubeviz. Thanks!